output/influxdb/config.go

```diff
@@ -38,15 +38,15 @@ import (
 type Config struct {
 	// Connection.
 	Addr             null.String        `json:"addr" envconfig:"K6_INFLUXDB_ADDR"`
-	Username         null.String        `json:"username,omitempty" envconfig:"K6_INFLUXDB_USERNAME"`
-	Password         null.String        `json:"password,omitempty" envconfig:"K6_INFLUXDB_PASSWORD"`
```

Comment:
I totally removed them but I think could be a good idea to maintain and make them compatible following the influx client's doc suggestion WDYT?.

>Use the form username:password for an authentication token. Example: my-user:my-password. Use an empty string ("") if the server doesn't require authentication.